Robot_Adapter/Convert/FromRobot/Elements/Panel.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Structure.MaterialFragments;
+using RobotOM;
+using System;
+using System.Collections.Generic;
+using BH.oM.Structure.Elements;
+using BH.oM.Structure.SectionProperties;
+using BH.oM.Structure.Constraints;
+using BH.oM.Structure.Offsets;
+using BH.oM.Adapters.Robot;
+using BH.Engine.Structure;
+using BH.oM.Geometry;
+using BH.Engine.Geometry;
+
+
+namespace BH.Adapter.Robot
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****           Public Methods                  ****/
+        /***************************************************/
+
+        public static bool FromRobotCheckFlipNormal(Vector normal, bool flip)
```

Comment:
`Convert` of a `Vector` into a Boolean 🤯 I do not like that in principle, because it is not converting a BHoM object to/from Toolkit-specific one. I would rather make it a `Modify` taking a Boolean and returning a Boolean, or even a `Compute`.

Reply:
I do get your point here, but as discussed in our call on teams, this is a larger issue. Will suggest a workshop around this for the coming milestone planning. Those methods are not the only ones that are breaking this principle in this/other toolkits.

Would prefer to get it to a cleaner/better state, but as discussed in the call, there are historic and UI reasons we ended up structuring this way.

Regarding convert, I both agree and do not agree. Robot treats the direction of the normal simply by a boolean value and a convention (always positive along the world axes, with prio to z over x over y). In this sense, this is actually converting one to one in terms of the function. Saying that, I do get your point that a boolean is not a vector.

Again, would be happy to discuss in a workshop, cause this can most surely be improved. Would prefer to not have it blocking this PR though, if you can live with it being fixed in the future! :)

Reply:
Having a workshop on that matter sounds good to me - your explanation makes a lot of sense but would be nice to inform and listen to others that potentially can be affected. Ideally, we could find a solution that would be less confusing, especially for the ones like me, coming from a different angle.

Based on the above, happy to approve this PR and looking forward to a follow-up session 👍